### ISSUE TITLE
Sound support

### ISSUE THREAD

**Issue by dlech** [ev3go/ev3dev#37]
It would be nice to have sound support. Tone/beep is easy to implement using the Linux input subsystem (evdev). Since we don't want to use cgo, we would have to call an external process (e.g. `aplay` for PCM playback.

**Comment by kortschak** [comment]
Yes, I was thinking that some sound support should be included, but an conflicted because using `aplay` is a trivial call to the `os/exec` package. The alternative is to use the approach used in the `syscall` package and pregenerate the C interface.

**Comment by kortschak** [comment]
The other thing to consider is that `evdev` is partially supported in `ev3go/ev3dev` to handle buttons. The work to extend that is a bit of a drudge, but I'm happy to look into it. Can you point me to some `C` or docs about how to use `evdev` for sound?

Note that this is also #11.

**Comment by dlech** [comment]
How about go code?

```go
package main

import (
	"log"

	"time"

	"github.com/jteeuwen/evdev"
)

func main() {
	dev, err := evdev.Open("/dev/input/by-path/platform-snd-legoev3-event")
	if err != nil {
		log.Fatal(err)
	}
	defer dev.Close()

	if !dev.EventTypes().Test(evdev.EvSound) {
		log.Fatal("Missing sound capability")
	}

	// play tone at 440Hz for 200ms
	ev := evdev.Event{Type: evdev.EvSound, Code: evdev.SndTone, Value: 440}
	dev.Outbox <- ev
	time.Sleep(200 * time.Millisecond)

	// play tone at 220Hz for 200ms
	ev = evdev.Event{Type: evdev.EvSound, Code: evdev.SndTone, Value: 220}
	dev.Outbox <- ev
	time.Sleep(200 * time.Millisecond)

	// stop tone playback
	ev = evdev.Event{Type: evdev.EvSound, Code: evdev.SndTone, Value: 0}
	dev.Outbox <- ev
	time.Sleep(200 * time.Millisecond)
}
```

**Comment by kortschak** [comment]
That's very straight forward - much simpler than the button presses. I won't use the library since it has some ugly practices that I would like to avoid (and aren't necessary for our use case).

**Comment by kortschak** [comment]
Closed by #40.
